### PR TITLE
ML: updates to containers list, OpenBLAS and other ML package versions

### PR DESCRIPTION
**Pull request by snadampal**
*Issue #, if available:*

*Description of changes:*

1. updated ML container urls
2. added OpenBLAS section to callout pthread and openmp versions
3. fixed few ML package names

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python.md [hunk 1]
```diff
@@ -237,14 +272,18 @@ $ conda env create -f graviton-example.yml
 pip install numpy
 pip install torch torchvision
 ```
+Please refer to the [Graviton PyTorch user guide](https://github.com/aws/aws-graviton-getting-started/blob/main/machinelearning/pytorch.md) for optimizing the PyTorch inference performance on Graviton.
```
Comment (AWSjswinney) [inline python.md:275]:
```suggestion
Please refer to the [Graviton PyTorch user guide](machinelearning/pytorch.md) for optimizing the PyTorch inference performance on Graviton.
```

File: python.md [hunk 2]
```diff
@@ -237,14 +272,18 @@ $ conda env create -f graviton-example.yml
...
+
 ### 3.2 TensorFlow
 
 ```
-pip install tensorflow-cpu-aws
+pip install tensorflow
 ```
+Please refer to the [Graviton TensorFlow user guide](https://github.com/aws/aws-graviton-getting-started/blob/main/machinelearning/tensorflow.md) for the recommended configuration and best practices.
```
Comment (AWSjswinney) [inline python.md:282]:
```suggestion
Please refer to the [Graviton TensorFlow user guide](machinelearning/tensorflow.md) for the recommended configuration and best practices.
```

File: python.md [hunk 3]
```diff
@@ -237,14 +272,18 @@ $ conda env create -f graviton-example.yml
...
+
 ### 3.3 DGL
 
-Make sure Pytorch is installed,  if not, follow [Pytorch installation steps](#41-pytorch)
+Make sure Pytorch is installed,  if not, follow [Pytorch installation steps](https://github.com/aws/aws-graviton-getting-started/blob/main/machinelearning/pytorch.md)
```
Comment (AWSjswinney) [inline python.md:286]:
```suggestion
Make sure Pytorch is installed,  if not, follow [Pytorch installation steps](machinelearning/pytorch.md)
```

File: containers.md [hunk 4]
```diff
@@ -31,9 +31,8 @@ We have compiled a list of popular software within the container ecosystem that
...
-| Tensorflow serving            | 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-inference-graviton:2.7.0-cpu-py38-ubuntu20.04-e3-v1.0      |                                                                                                         |
-| PyTorch                       | https://hub.docker.com/r/armswdev/pytorch-arm-neoverse                                                             | Use tags with *-openblas* for performance reasons until confirmed otherwise                             |
+| TensorFlow with TensorFlow Serving                     | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference-graviton:2.13.0-cpu-py310-ubuntu20.04-ec2                                                          |   refer to [Graviton TensorFlow user guide](https://github.com/aws/aws-graviton-getting-started/blob/main/machinelearning/tensorflow.md) for how to use.                                                                                                      |
```
Comment (AWSjswinney) [inline containers.md:34]:
```suggestion
| TensorFlow with TensorFlow Serving                     | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference-graviton:2.13.0-cpu-py310-ubuntu20.04-ec2                                                          |   refer to [Graviton TensorFlow user guide](machinelearning/tensorflow.md) for how to use.                                                                                                      |
```

File: containers.md [hunk 5]
```diff
@@ -31,9 +31,8 @@ We have compiled a list of popular software within the container ecosystem that
...
+| PyTorch with TorchServe                      | 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference-graviton:2.1.0-cpu-py310-ubuntu20.04-ec2                                                            | refer to [Graviton PyTorch user guide](https://github.com/aws/aws-graviton-getting-started/blob/main/machinelearning/pytorch.md) for how to use                            |
```
Comment (AWSjswinney) [inline containers.md:35]:
```suggestion
| PyTorch with TorchServe                      | 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference-graviton:2.1.0-cpu-py310-ubuntu20.04-ec2                                                            | refer to [Graviton PyTorch user guide](machinelearning/pytorch.md) for how to use                            |
```

**Review by AWSjswinney — CHANGES_REQUESTED** [review]
The links to other pages on the GSG can be relative links.

**Comment by snadampal** [comment]
@AWSjswinney , thank you! addressed your feedback.

**Review by AWSjswinney — APPROVED** [review]

**Pull request merged**